quantum/rgb_matrix/animations/pixel_fractal_anim.h

```diff
@@ -25,7 +25,7 @@ static bool PIXEL_FRACTAL(effect_params_t* params) {
     static bool led[MATRIX_ROWS][MATRIX_COLS];
 
     static uint32_t wait_timer = 0;
-    if (wait_timer > g_rgb_timer) {
+    if (wait_timer > timer_read32()) {
```

Comment:
```suggestion
    if (wait_timer > sync_timer_read32()) {
```

Reply:
Hi @drashna, this doesn't fix the issue since we are using `SPLIT_KEYBOARD` and there are switch macros to use a different implementation (use a volatile variable. See below) for split keyboards in `sync_timer.h` and `sync_timer.c`
```
#if defined(SPLIT_KEYBOARD) && !defined(DISABLE_SYNC_TIMER)
volatile int32_t sync_timer_ms;
```

Reply:
My biggest issue with the change is that the animations do require syncing on split, and that's why the sync_timer stuff is used here (albeit indirectly).

Changing that to just timer_read may cause the animations to become desynced.

Reply:
Hi @drashna. I think I got your point. I am going to work finding the best solution for this issue. Thanks!